src/components/Person.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import '../App.scss';
+
+const Person = ({ name, age, sex, isMarried, partnerName }) => {
+  const partner = sex === 'f' ? 'husband' : 'wife';
+
+  return (
+    <section className="person">
+      <h2 className="person__name">{`My name is ${name}`}</h2>
+      <p className="person__age">{age ? `I am ${age}` : null}</p>
```

Comment:
You can use `&&`(and) operator here with your `age` property and based on that use conditional rendering for the whole `<p>` tag, not only text inside